docs/guides/runtime-options-overview.mdx

```diff
@@ -18,288 +19,308 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
 ```
 </Admonition>
 
-<Admonition type="info" title="Attention">
-    This section focuses on Qiskit Runtime primitive options. While most of the `primitives` interface is common across implementations, most options are not. Consult the
-    corresponding API references for information about the `qiskit.primitives` and `qiskit_aer.primitives` options.
-</Admonition>
+## Overview
 
-## V2 changes
-Options are specified differently in the V2 primitives in these ways:
+<span id="options-structure"></span>
+### Structure
 
-- `SamplerV2` and `EstimatorV2` now have separate options classes. You can see the available options and update option values during or after primitive initialization.
-- Instead of the `set_options()` method, V2 primitive options have the `update()` method that applies changes to the `options` attribute.
-- If you do not specify a value for an option, it is given a special value of `Unset` and the server defaults are used.
-- For V2 primitives, the `options` attribute is the `dataclass` Python type.  You can use the built-in `asdict` method to convert it to a dictionary.
+When calling the primitives, you can pass in options by using an options class or a dictionary. Commonly used options, such as `resilience_level`, are at the first level. Other options are grouped into different categories, such as `execution`. See the [Set primitive options](specify-runtime-options) topic for full details.
 
-## Instantiate the Options class (V1)
+<span id="options-defaults"></span>
+### Defaults
 
-In the example below, we create an instance of the `Options` class. `optimization_level` is a first-level option and can be passed as an input parameter. Options related to the execution environment are passed using the `environment` parameter.
+For V2 primitives, if you do not specify a value for an option, it is given a special value of `Unset` and the server default value is used. Thus, the default value will be the same regardless of your code version.
+For V1 primitives, if you do not specify a value for an option, the Qiskit Runtime client's default value is used. In V1, the default value is dependant on the code version.
 
-```python
-from qiskit_ibm_runtime import Options
+The tables in the [Options classes summary](#options-classes) section lists the default values.
 
-options = Options(optimization_level=1, environment={"log_level": "INFO"})
-```
+<span id="options-precedence"></span>
+## Set options
```

Comment:
This entire section applies only to V2 primitives. We should have one for V1, or at least make it explicit.

Reply:
I think I fixed this.